src/components/common/ColonyDecisions/DraftDecisionItem/DraftDecisionItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { Placement } from '@popperjs/core';
+
+import ListItem, { ListItemStatus } from '~shared/ListItem';
+import UserAvatar from '~shared/UserAvatar';
+import { Decision } from '~types';
+import { MotionState } from '~utils/colonyMotions';
+import { useAppContext, useColonyContext } from '~hooks';
+
+import { PREVIEW_ROUTE_SUFFIX } from '../DecisionDialog';
+import DraftDecisionActions from './DraftDecisionActions';
+
+import styles from './DraftDecisionItem.css';
+
+const displayName = 'common.ColonyDecisions.DraftDecisionItem';
+
+const avatarPopoverOptions = {
+  showArrow: false,
+  placement: 'bottom-start' as Placement,
+  modifiers: [
+    {
+      name: 'offset',
+      options: {
+        offset: [-20, 30],
+      },
+    },
+  ],
+};
+
+export interface DraftDecisionItemProps {
+  decision: Decision;
+}
+
+const DraftDecisionItem = ({ decision }: DraftDecisionItemProps) => {
+  const navigate = useNavigate();
+  const { colony } = useColonyContext();
+  const { pathname } = useLocation();
+  const { user } = useAppContext();
+  const walletAddress = user?.walletAddress || '';
+
+  const redirectToPreview = () =>
+    navigate(`${pathname}${PREVIEW_ROUTE_SUFFIX}`);
+
+  if (!decision) {
+    return null;
+  }
+
+  return (
+    <div className={styles.draftDecision}>
+      <ListItem
```

Comment:
This `ListItem` component is an initial attempt to extract the view of the base "ListItem" from the accompanying logic associated with `ActionListItem`. I expect it will be modified should it become the base component of `ActionListItem` also (which could really do with a refactor), but this is a start.

Reply:
Makes sense, could you please open a issue to track that ?

Reply:
Worked on this here #229